src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -150,5 +156,10 @@ public void should_return_a_403_when_the_assignee_does_not_have_execute_or_own_p
                   equalTo("User did not have sufficient permissions to assign task with id: " + taskId));
     }
 
+    @After
+    public void cleanUp() {
+        camundaApiActions.post(ENDPOINT_COMPLETE_TASK, taskId,
+                               new Headers(authorizationHeadersProvider.getServiceAuthorizationHeader()));
```

Comment:
could we have something like:
instead of wrapping it in new Headers `authorizationHeadersProvider.getServiceAuthorizationHeadersOnly()`